### PR TITLE
Fix Taming skill max level

### PR DESCRIPTION
**Pull request by Peng1104**
Fix one of the bugs described in #2212

### CODE REVIEW

File: src/stats/skills.js [hunk 1]
```diff
@@ -12,10 +12,7 @@ async function getLevels(userProfile, profileMembers, hypixelProfile, levelCaps)
     }, 0);
 
     Object.assign(skillLevels, {
-      taming: getLevelByXp(SKILL.SKILL_TAMING, {
-        skill: "taming",
-        cap: Math.max(hypixelProfile.achievements.skyblock_domesticator, 50),
-      }),
+      taming: getLevelByXp(SKILL.SKILL_TAMING, {skill: "taming", cap: levelCaps.taming }),
```
Comment (metalcupcake5) [inline src/stats/skills.js:15]: raising taming cap requires submitting pets to george, so this must be determined by achievement level

**Review by metalcupcake5 — CHANGES_REQUESTED** [review]

File: src/stats/skills.js (hunk 1 above)
Comment (Peng1104) [reply src/stats/skills.js:15]: The problem is that this don't work, since the update where you can give the pets to George in a random order.

Comment (Peng1104) [reply src/stats/skills.js:15]: The Domesticator achievement only give information about the 5/10/15/20 and 25 taming levels, nothing to do whit George (https://wiki.hypixel.net/Achievements)

Comment (DuckySoLucky) [reply src/stats/skills.js:15]: yeah but it's still a lot better than no cap, now it won't be accurate at all, imo this PR shouldn't be merged

Comment (Peng1104) [reply src/stats/skills.js:15]: @DuckySoLucky then way is farming so? As farming also has a skill cap, you need to get gold in a farming collection and then buy the extra farming level whit a gold medal. There is no check for that you have it on the code but It dosent work, the same for taming the website says I am capped but I am not.

Comment (Peng1104) [reply src/stats/skills.js:15]: > @DuckySoLucky then way is farming so? As farming also has a skill cap, you need to get gold in a farming collection and then buy the extra farming level whit a gold medal. There is no check for that you have it on the code but It dosent work, the same for taming the website says I am capped but I am not.

never mind farming is working fine but taming is not, why not set cap 50 directly then?

Comment (DuckySoLucky) [reply src/stats/skills.js:15]: > @DuckySoLucky then way is farming so? As farming also has a skill cap, you need to get gold in a farming collection and then buy the extra farming level whit a gold medal. There is no check for that you have it on the code but It dosent work, the same for taming the website says I am capped but I am not.

~~what are you talking about? farming works?~~
nvm just saw msg below

Comment (DuckySoLucky) [reply src/stats/skills.js:15]: > > @DuckySoLucky then way is farming so? As farming also has a skill cap, you need to get gold in a farming collection and then buy the extra farming level whit a gold medal. There is no check for that you have it on the code but It dosent work, the same for taming the website says I am capped but I am not.
> 
> never mind farming is working fine but taming is not, why not set cap 50 directly then?

cuz using the achievement is better than settings it to just 50 or 60, and also we at SkyHelper do it that way as well so we should stay consistent in my opinion

Comment (DuckySoLucky) [reply src/stats/skills.js:15]: > > @DuckySoLucky then way is farming so? As farming also has a skill cap, you need to get gold in a farming collection and then buy the extra farming level whit a gold medal. There is no check for that you have it on the code but It dosent work, the same for taming the website says I am capped but I am not.
> 
> never mind farming is working fine but taming is not, why not set cap 50 directly then?

cuz using the achievement is better than settings it to just 50 or 60, and also we at SkyHelper do it that way as well so we should stay consistent in my opinion

Comment (Peng1104) [reply src/stats/skills.js:15]: Ok I will make the change back but the Max level for taming need to be 60 or else you will have a gold bar for not a absolute max level

Comment (Peng1104) [reply src/stats/skills.js:15]: Sorry for my confusion now that I had a deeper look in the code I got my mistake, I was assuming that level Cap == Max Level, adding a taming max level on the constants already fixed the error no need to change this part.

Comment (DuckySoLucky) [inline src/stats/skills.js:15]: that's gonna just remove cap completely?

Comment (DuckySoLucky) [inline src/stats/skills.js:15]: that's gonna just remove cap completely?

**Review by DuckySoLucky — CHANGES_REQUESTED** [review]

File: src/stats/skills.js (hunk 1 above)
Comment (Peng1104) [inline src/stats/skills.js:15]: > that's gonna just remove cap completely?

No just moved the cap for the "levelCaps" for consistency, same way farming / carpentry is done (Line 79)

**Review by DuckySoLucky — APPROVED** [review]
lgtm

**Review by metalcupcake5 — APPROVED** [review]

**Review by Shiiyu — CHANGES_REQUESTED** [review]
Prettier has failed on src/stats/skills.js

**Pull request closed without merging**